tests/conftest.py

```diff
@@ -8,15 +8,16 @@
 impala_ldap = {
         'type': 'impala',
         'threads': 5,
+        'dbname': 'dbttest',
         'schema': 'dbttest',
         'host': 'coordinator-dbt-impala.dw-ciadev.cna2-sx9y.cloudera.site',
         'http_path': 'cliservice',
         'port': 443,
         'auth_type': 'ldap',
         'use_http_transport': True,
         'use_ssl': True,
-        'user': os.getenv('IMPALA_USER'),
```

Comment:
was this intended

Reply:
good catch. 
unintended consequence of revering.

Reply:
fixed this